src/test/Components/CreateImageWizard/CreateImageWizard.compliance.test.js

```diff
@@ -84,36 +87,66 @@ describe('Step Compliance', () => {
     await clickNext();
 
     // Now we should be in the Compliance step
-    await screen.findByRole('heading', { name: /OpenSCAP Compliance/i });
-    expect(
-      await screen.findByRole('radio', { name: /do not add a profile/i })
-    ).toBeChecked();
+    await screen.findByRole('heading', { name: /OpenSCAP/i });
+
     await user.click(
-      await screen.findByRole('radio', { name: 'Add a profile' })
+      screen.getByRole('textbox', { name: /select a profile/i })
     );
-    expect(
-      await screen.findByRole('radio', { name: 'Add a profile' })
-    ).toBeChecked();
+    await user.click(screen.getByText(/none/i));
+
+    // check that the FSC contains a /tmp partition
+    await clickNext();
+    await screen.findByRole('heading', { name: /File system configuration/i });
+
+    // check that the Packages contain a nftable package
+    await clickNext();
+    await screen.findByRole('heading', {
+      name: /Additional Red Hat packages/i,
+    });
+  });
+
+  test('create an image with an oscap profile', async () => {
+    await setup();
+
+    // select aws as upload destination
+    await user.click(await screen.findByTestId('upload-aws'));
+    await clickNext();
+
+    // aws step
     await user.click(
-      await screen.findByRole('textbox', { name: /select a profile/i })
+      screen.getByRole('radio', { name: /manually enter an account id\./i })
     );
+    await user.type(screen.getByTestId('aws-account-id'), '012345678901');
+
+    await clickNext();
+    // skip registration
+    await user.click(screen.getByLabelText('Register later'));
+    await clickNext();
+
+    // Now we should be in the Compliance step
+    await screen.findByRole('heading', { name: /OpenSCAP/i });
+
     await user.click(
-      await screen.findByRole('option', {
-        name: /xccdf_org\.ssgproject\.content_profile_cis_workstation/i,
+      screen.getByRole('textbox', {
+        name: /select a profile/i,
       })
     );
 
+    await user.click(
+      await screen.findByText(
+        /cis red hat enterprise linux 8 benchmark for level 1 - workstation/i
+      )
+    );
     // check that the FSC contains a /tmp partition
```

Comment:
The tests don't check the `/tmp` mount point. Which they probably should in this place. 

The structure of the test could be following:
1. get to the OpenSCAP step
2. select a profile
3. click Next
4. check if there is a mount point which is defined in the profile
5. click Next
6. check if there is/are package/s added based on the profile

Reply:
I covered that test in row 149   // check that the FSC contains a /tmp partition
    await clickNext();
    await screen.findByRole('heading', { name: /File system configuration/i });
    await screen.findByText(/tmp/i);